%% This is file `elsarticle-template-1-num.tex',
%%
%% Copyright 2009 Elsevier Ltd
%%
%% This file is part of the 'Elsarticle Bundle'.
%% ---------------------------------------------
%%
%% It may be distributed under the conditions of the LaTeX Project Public
%% License, either version 1.2 of this license or (at your option) any
%% later version.  The latest version of this license is in
%%    http://www.latex-project.org/lppl.txt
%% and version 1.2 or later is part of all distributions of LaTeX
%% version 1999/12/01 or later.
%%
%% Template article for Elsevier's document class `elsarticle'
%% with numbered style bibliographic references
%%
%% $Id: elsarticle-template-1-num.tex 149 2009-10-08 05:01:15Z rishi $
%% $URL: http://lenova.river-valley.com/svn/elsbst/trunk/elsarticle-template-1-num.tex $
%%
\documentclass[preprint,final,times,12pt,3p]{elsarticle}

%% Use the option review to obtain double line spacing
%% \documentclass[preprint,review,12pt]{elsarticle}

%% Use the options 1p,twocolumn; 3p; 3p,twocolumn; 5p; or 5p,twocolumn
%% for a journal layout:
%% \documentclass[final,1p,times]{elsarticle}
%% \documentclass[final,1p,times,twocolumn]{elsarticle}
%% \documentclass[final,3p,times]{elsarticle}
%% \documentclass[final,3p,times,twocolumn]{elsarticle}
%% \documentclass[final,5p,times]{elsarticle}
%% \documentclass[final,5p,times,twocolumn]{elsarticle}


%%\usepackage[left=1in, right=1in, top=1in, bottom=1in]{geometry}
\usepackage{array}
\usepackage{multirow}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsthm}
\usepackage{amsmath}
\usepackage{lineno}
\usepackage{setspace}
\usepackage{pdflscape}
\usepackage{natbib}
\usepackage{listings}
\usepackage{lstbayes}
\usepackage{longtable}

\journal{Ecological Modelling}
\setcitestyle{authoryear,round,semicolon,sort}
\bibliographystyle{apalike}
\begin{document}
\begin{frontmatter}

\title{Leveraging citizen science to assess richness, diversity, and abundance}

\author[DEE]{Tim M. Szewczyk}
\author[DEE]{Cleo Bertelsmeier}
\author[DEE]{Tanja Schwander}

\address[DEE]{Department of Ecology and Evolution, University of Lausanne}


\begin{abstract}
Citizen science is a key resource in overcoming the logistical challenges of monitoring biodiversity as the climate continues to change. While datasets collected by groups of volunteers typically have biases, recent methodological and technological advances provide approaches for accounting for these issues, particularly in the context of modelling species distributions and diversity. Specifically, data integration techniques allow for the combination of scientifically collected datasets with presence-only datasets like those often created by citizen science initiatives. Here, we use a hierarchical Bayesian framework to integrate a set of ant presences collected by citizen scientists in the canton of Vaud, Switzerland with concurrently collected local ant colony density data in the same region. The community-level Poisson point process model included species-specific responses to the local (1.2 m$^2$) and regional (1 km$^2$) environment, with the presence-only samples incorporated at the regional scale to predict local and regional ant communities. We found that, regionally, richness peaked near 1000 m, with distinct clustering of low elevation and montane communities. Using both datasets, regional Shannon diversity was predicted to decline with elevation, while regional abundance increased. There was minimal elevational pattern in local richness, diversity, and abundance. At low elevations, local plots varied both in total abundance and species composition, while at higher elevations, the species composition was less variable. The citizen science dataset showed a general tendency toward under-representation of species, and heavy spatial sampling bias. Nonetheless, the inclusion of the citizen science data improved predictions of novel local communities, and also reduced susceptibility to over-fitting. Additionally, the citizen science dataset included many rare species not detected in the structured abundance dataset. The model described here illustrates a framework for capitalizing on the efforts of citizen scientists to better understand the patterns and distribution of biodiversity.
\end{abstract}

\begin{keyword}
altitude; ants; Bayesian; elevation; Formicidae; Poisson point process
\end{keyword}

\end{frontmatter}
\linenumbers



\section{Introduction}
\label{S:1}

% Growth of citizen science
Citizen science is one solution to many of the logistical complications that hinder ecological research at broad scales. The many volunteer collectors can sample ecological systems across regional, continental, and even global extents \citep{Theobald2015}, and include private land that is otherwise inaccessible to researchers \citep{Pernat2020}. Accordingly, citizen science initiatives have become increasingly common, with applications ranging from monitoring the spread of invasive rabbits in Australia \citep{Roy-Dufresne2019} to identifying impacts of hunting on game species' life history traits in the Amazon \citep{ElBizri2020}. Samples collected by citizen scientists represent a sizable component of many ecological datasets \citep{Poisson2020}. While projects to-date show similar taxonomic biases as broader ecological research \citep{Theobald2015,Troudet2017}, citizen science could be particularly useful in overcoming the knowledge shortfall of insect distributions, since specimens can be collected and easily returned for expert identification or genetic analysis \citep{Ward2014,Pernat2020}.

% Value and potential shortcomings
The efforts of citizen scientists represent a valuable resource for understanding and monitoring the distribution of biodiversity on a rapidly changing planet. Unfortunately, despite the explosion in citizen science initiatives, as few as 12\% are associated with peer-reviewed publications \citep{Theobald2015}. Resulting datasets do have limitations, even with simple designs such as collecting unstructured occurrences for use in species distribution models \citep{Theobald2015,Steen2019,Duan2020,Henckel2020}. The typically decentralized nature of the data collection may require additional quality checks and filters, consideration of potential errors in identifications or geo-location, and handling of non-random sampling. However, advances in modelling provide viable solutions to compensate for these pitfalls. For example, after accounting for such issues as spatial bias \citep{Isaac2014,Steen2019,Johnston2020,Robinson2020}, the performance of species distribution models fit with citizen science data can be equivalent or superior to those fit with scientifically collected data \citep{Steen2019,Sumner2019,Henckel2020}. 

% Modelling richness with presence-only data: iSDMs, jSDMs, MRMs, mSDMs
Survey-type citizen science projects often collect a wide set of species within a taxon, making them appealing for estimating taxon-level patterns of richness. A variety of methods have been developed for modelling richness, broadly categorized into those that model richness directly as a function of environmental variables, and those that grew from modelling individual species distributions \citep{Dubuis2011,Guisan2011,Calabrese2014,Biber2019}. For the latter, a simple approach stacks independent species distribution models, while more complex approaches model distributions jointly or in a hierarchical framework \citep{Caradima2019}. Alternatively, multi-species occupancy models explicitly account for imperfect detection, predicting the underlying occupancy state and probability of occurrence for each species \citep{Frishkoff2019,Guillera-Arroita2019,Szewczyk2018,Devarajan2020}. However, occupancy models require presence-absence data from multiple sampling bouts to estimate detection probabilities, and consequently their use is highly contingent on the study design.

% Methods for merging datasets
Recent data integration techniques allow researchers to capitalize on the strengths of unstructured presence-only datasets, such as those collected by citizen scientists, while overcoming their limitations by combining them with structured samples collected by researchers \citep{Isaac2019,Miller2019}. These methods aim to exploit the strengths of each dataset. For example, presence-only data often detecting more rare species, while datasets collected in a structured manner better represent community composition and abundance \citep{Steen2019,Henckel2020,Pernat2020}. Integrating disparate datasets often leads to improved model performance \citep{Dorazio2014b,Fithian2015,Koshkina2017a}. Bayesian methods naturally and flexibly accommodate the incorporation of multiple datasets \citep{Clark2005,BeckEtAl2012,Szewczyk2018}, either by estimating a joint likelihood for all observations, or by incorporating the presence-only dataset via predictors or prior distributions \citep{Fletcher2019,Isaac2019,Miller2019}. 

% Poisson point process models to unite spatial scales and accommodate density/abundance data
One natural choice for modelling species distributions are inhomogenous Poisson Point Process Models (PPMs), which model continuous, spatially varying intensities that produce Poisson-distributed occurrences \citep{Renner2013a,Renner2015}. PPMs are an especially appealing option for combining different datasets. Observations collected with different methods can be modelled using appropriate sampling submodels based on the same underlying intensities \citep{Fithian2015,Hefley2016,Koshkina2017a,Fletcher2019,Renner2019}, and the intensities can be integrated to the appropriate resolution, allowing patterns and processes to be linked across spatial scales \citep{Keil2014,Hefley2016}.

% Mission statement
Here, we combine presences of ants collected in a citizen science project in the canton of Vaud, Switzerland with local abundance data from a concurrent structured sampling effort. In a hierarchical Bayesian framework, this community-level PPM jointly models local ant colony density, with additional presence-only information integrated at a regional scale, while accounting for taxonomy and correlated residuals among species resulting from unmeasured variables or biotic interactions. We use this model to predict the patterns of ant colony density, richness, diversity, and community structure across the landscape at local and regional scales while incorporating uncertainty in species compositions, and to evaluate the support for hypothesized environmental drivers across spatial scales. We compare inferences from the integrated model with those from only the structured dataset, and also assess the differences in observed communities from each sampling method.





\section{Methods}
\label{S:2}
\subsection{Study region \& sampling design}
This study focuses on ant species in the canton of Vaud in western Switzerland (46.2–47.0$^{\circ}$N, 6.1–7.2$^{\circ}$E), a topographically heterogeneous region (372–3210 m) composed of the Swiss plateau centrally, the Jura mountains in the west, and the Alps in the east (Fig. \ref{fig:VD_map}). The hilly central plateau is dominated by anthropogenic land use, including cropland, vineyards, pastures, and urban zones, with patches of predominantly deciduous forest. The mountain ranges rise steeply from approximately 1000m and span the forested montane and subalpine zones through the alpine zone, with widespread cattle grazing \citep{Delarze2015,Gago-Silva2017,Beck2017}. 

\begin{figure}
	\centering\includegraphics[width=3.5in]{../../../ms/1_Ecography/1/figs/map_VD+inset.png}
	\caption{\label{fig:VD_map} Map of the canton of Vaud, Switzerland. Samples consisted of presence-only data (blue points) aggregated to a 1 km$^2$ grid (light gray) and structured abundance data collected at 1 km$^2$ long-term biodiversity monitoring sites (red squares). }
\end{figure}

% Methods for citizen science data
Ants were collected during the summer of 2019 in two simultaneous efforts (Fig. \ref{fig:VD_map}), comprising a citizen science presence-only dataset and a structured abundance dataset. The citizen science project (Opération Fourmis: https://wp.unil.ch/fourmisvaud/) was organized to survey the ant fauna within the canton of Vaud. Vials of ethanol were distributed to interested participants, who were encouraged to explore under rocks, on bark, inside twigs, and in downed wood, and asked to collect approximately 10 ant workers from a single colony for each vial. An online map was updated periodically to informally highlight data-sparse areas. Collectors returned the vials along with the collection date and the locality of the sample, including geolocation and a brief habitat description. Additionally, several guided collecting field trips were organized throughout the season. For these presence-only data, we discretized the landscape into a 1 km$^2$ grid (3,558 km$^2$) and tallied the number of occurrences for each species in each cell. 

% Methods for structured samples
For the structured abundance dataset, local ant densities were estimated within 44 sites (1 km$^2$ each), with standardized effort across sites. Thirty-nine sites were long-term governmental biodiversity monitoring sites, and as such were arranged on a regular grid with approximately 5-7 km between adjacent sites. Five additional sites were pre-established monitoring sites by the University of Lausanne. At each site, ant densities were characterized by 25 plots, distributed among 15 habitat types \citep{Gago-Silva2017} in approximate proportion to the abundance of each habitat, where each habitat present within the site was represented by at least one plot (Table \ref{table:landcover}). Inaccessible areas (e.g., cliffs, water, property beyond Vaud) were excluded, resulting in several sites with areas less than 1 km$^2$, and the number of plots reduced proportionally (1059 total plots). 

Each sampling plot consisted of a 2 m radius circle, with soil temperature recorded in the center approximately 6 cm deep, and vegetation characterized according to Braun-Blanquet coverage classes for grass, forb, shrub, litter, bare ground, and moss within the plot \citep{Douglas1978}. Six flags were evenly spaced around the circumference. Within $\sim$ 25 cm of each flag (total surface area $\sim$ 1.2 m$^2$), we searched for ant colonies within any downed wood or stumps, under large rocks, and in 2 L of soil, litter, and small rocks using 18 cm Hori Hori gardening knives. We haphazardly collected 10 workers from each colony, placing them directly into vials of ethanol. Within each plot, all trees $\geq$3 cm diameter at breast height were also inspected for ant workers which were collected regardless of whether or not a nest was identifiable. Lastly, transect lines were mapped \emph{a priori}, distributed proportionally among habitat types and totalling 2 km. Transects were surveyed at a moderate pace, with workers collected from all permanent above-ground mounds within 2 m of the transect line. Because of the differing sampling methods, the tree and transect collections were incorporated into the presence-only dataset. 

Thus, the two datasets characterize the regional ant fauna in distinct ways. The presence-only dataset was spatially expansive and included free investigation of subjectively suitable habitats, and was therefore expected to include more rare species. However, collections were also expected to be spatially and taxonomically biased, with more effort near human-dominant areas and toward larger, more obvious, or more anthropophilic species \citep{Ward2014, Troudet2017}. In contrast, the structured abundance dataset was collected with uniform sampling effort, with a design aimed to produce representative samples of colony density within each site. This dataset was thus expected to better characterize the ground-nesting ant community structure. 

% Identification and storage
All ants were identified to species or species group based on morphology by taxonomists. Four genera (\emph{Camponotus}, \emph{Tapinoma}, \emph{Temnothorax}, \emph{Tetramorium}) were identified based on mitochondrial DNA due to uncertainties in morphological identifications in these groups. Specimens are stored at the Natural History Museum of Lausanne in Lausanne, Switzerland. 


\subsection{Environmental variables}
Environmental variables were selected based on theoretical or empirical support in the literature as likely drivers of ant distributions, richness, and diversity, \citep{Bishop2017,Liu2018,Szewczyk2018,Longino2019,Uhey2020}. At a regional (1 km$^2$) scale, we included growing degree days from ENVIREM as well as its square \citep{Title2018}, annual precipitation from CHELSA \citep{Karger2017}, net primary productivity as the average values across years 2010–2019 (MODIS: MOD17A3), the Shannon diversity of the 15 land cover types based on proportional composition \citep{Gago-Silva2017}, the coverage of edge, forest, and agricultural habitats \citep{Gago-Silva2017}, the average north-facing aspect calculated as the cosine of the aspect in degrees based on the ASTER digital elevation model \citep{Tachikawa2011}, the log-transformed total length of roads \citep{OpenStreetMap}, and the log-transformed total perimeter of buildings \citep{OpenStreetMap}. All variables were summarised as the mean or total value within each 1 km$^2$ grid cell and within each 1 km$^2$ structured site.

Local variables were collected in the field at each structured sampling plot as described above. We included relative soil temperature as a measurement of local variation in temperature, where the recorded temperatures among plots were z-transformed within each site. Canopy was classified as open, closed, or mixed according to land cover type (Table \ref{table:landcover})). Open habitats were categorized as pasture, crop, or other according to field observations. Finally, local productivity was quantified by using the midpoint of the range of percent cover values in each Braun-Blanquet category and summing across grass, forb, and shrub in each plot \citep{Douglas1978,McCain2018,Szewczyk2018}.

Variable processing and summarising was performed in R 3.6.3 \citep{R-3-6-3}, with spatial computations in the R packages \emph{raster 3.3.7} and \emph{sf 0.9.4}. All variables were z-transformed after summarising to the appropriate spatial scale to improve model behavior \citep{Carpenter2017}.


\subsection{Model overview}
To leverage the strengths of each dataset, we used a community-level hierarchical inhomogenous Poisson point process model (PPM) fit in a Bayesian framework. Inhomogenous Poisson PPMs assume that the distribution of occurrences is dependent on the variation in local intensity, which may be observed imperfectly resulting in a thinned point process \citep{Warton2010,Baddeley2015,Fithian2015}. One key benefit of PPMs is that the underlying latent intensity is continuous in space, and can be integrated to arbitrary spatial resolutions \citep{Renner2015,Hefley2016,Koshkina2017a,Fletcher2019}. Following the structure of the sampling design, we modelled the expected intensity of each species at two resolutions (regional: 1 km$^2$, local: 1.2 m$^2$), representing the area of the sampling sites and the area of the sampling plots respectively. We modelled local intensities as a function of species' responses to the local and regional environment, with the structured abundance data incorporated at the local scale, and the presence-only data incorporated at the regional scale. The sampling submodel for each data type reflects the collection methods \citep{Isaac2014,Hefley2016,Fletcher2019,Miller2019}.


\subsection{Model structure}
The hierarchical PPM integrates the two datasets, \textbf{W} (presence-only; resolution: 1 km$^2$) and \textbf{Y} (structured local abundance; 1.2 m$^2$ plots within 1 km$^2$ sites) to predict ant communities across the landscape. Species' responses to the regional environment are informed by both \textbf{W} and \textbf{Y}, while responses to the local environment are informed by \textbf{Y}. Similarly, \textbf{W} helps to identify overall relative abundance among species, while \textbf{Y} is a direct measurement of local abundance. We assume that local colony density for each species is a function of local and regional environmental conditions, with potential phylogenetic conservatism among species-specific responses and residual correlation among species.

The structured abundances, \textbf{Y}, are observed at the local scale. For each species $s=1 \dots S$  at plot $i$, the number of observed colonies $Y_{is}$ follows a generalized Poisson distribution with the latent colony intensity $\lambda_{is}$ and dispersion term $\theta$ to account for overdispersion \citep{Consul1992,Ntzoufras2005,Isaac2019,Miller2019}. The local intensity $\lambda_{is}$ is a function of the local environment and the regional intensity of species $s$ at the encompassing 1 km$^2$ site $j$:
    \begin{equation}
        Y_{is} \sim GPoisson(\lambda_{is}, \theta) 
        \label{eq:Y_GP}
    \end{equation}
    \begin{equation}
        log(\lambda_{is}) = a_s V_i + \gamma_s \zeta_i + log(h\Lambda_{js}) 
        \label{eq:lambda}
    \end{equation}
where \textbf{V} is a matrix of local environmental covariates, $a_s$ is a vector of species-specific responses, $\zeta_i$ is a latent factor representing unmeasured local environmental variables or biotic interactions with $\gamma_s$ as a species-specific latent factor coefficient constrained between $-1$ and $1$ \citep{Ovaskainen2016,Caradima2019,Tobler2019}, $h$ is a constant scaling factor representing the proportion of site $j$ sampled by plot $i$ (1.2 $m^2 /$ 1 km$^2$ = 1.2e-6), and $\Lambda_{js}$ is the regional intensity of species $s$ at site $j$. Thus, $log(h\Lambda_{js})$ functions as a site-level intercept, determining the baseline expected intensity at each plot within a site \citep{Yamaura2016,Miller2019}.

The presence-only data, \textbf{W}, are observed as detections at a regional scale. The detections of species $s=1 \dots S$ within each 1 km$^2$ cell $k$ follow a multinomial distribution such that:
    \begin{equation}
        \{W_{ks}\}_{s=1}^{S} \sim Multinomial(\sum_{s=1}^{S} W_{ks} , \varsigma_k(\{log(\Lambda_{ks})D_s\}_{s=1}^{S})) 
        \label{eq:W_Mn}
    \end{equation}
where $\{W_{ks}\}_{s=1}^{S}$ is a vector of length $S$ with the number of detections of each species within cell $k$, $\sum_{s=1}^{S} W_{ks}$ is the total number of detections within the cell, $D_s$ is the proportional detection bias for each species which accounts for taxonomic bias in effort, and $\varsigma_k(\cdot)$ is the softmax function which converts the weighted intensities to probabilities that sum to 1 within each cell. Thus, for species $s$ in cell $k$, the observed count $W_{ks}$ is expected to be higher if the species has high relative abundance ($\Lambda_{ks}$ is larger relative to other species), more samples were collected ($\sum_{s=1}^{S} W_{ks}$ is larger), or if species $s$ is over-represented in \textbf{W} relative to \textbf{Y} ($D_s$ is larger). The presence-only counts thus inform relative patterns across the landscape, while accounting for spatial and taxonomic bias in sampling intensity \citep{Isaac2014}.

While \textbf{W} and \textbf{Y} are modelled with distinct sampling submodels, both $\Lambda_{js}$ and $\Lambda_{ks}$ represent the latent intensity of species $s$ at an identical 1 km$^2$ regional scale. Consequently, the regional intensity $\Lambda_{(j,k)s}$ is a shared parameter that links the two datasets \citep{Hefley2016,Isaac2019,Miller2019}. The ecological processes driving variation in regional intensity are thus independent of sampling method, and the intensities are modelled together in a single regression such that:
    \begin{equation}
        log(\Lambda_{(j,k)s}) = b_s X_{(j,k)} 
        \label{eq:LAMBDA}
    \end{equation}
where $b_s$ is a vector of species-specific slopes, and \textbf{X} is a matrix of regional environmental covariates. Thus, the magnitude and spatial variation of the 1 km$^2$ intensity $\Lambda_{(j,k)s}$ is informed by $Y_{is}$ (Eq. \ref{eq:Y_GP}-\ref{eq:lambda}), with the spatial variation further informed by $W_{ks}$ (Eq. \ref{eq:W_Mn}).

The slopes $a_s$ and $b_s$ are species-specific responses at local and regional scales, respectively, and are distributed about genus-level means $A_g$ and $B_g$ with standard deviations $\sigma_a$ and $\sigma_b$. The genus-level means are in turn distributed about aggregate means, $\alpha$ and $\beta$, with correlation matrices $\Sigma_A$ and $\Sigma_B$, which reflect the overall responses of the ant community to environmental variables at each resolution while accounting for relatedness at the genus level \citep{Hadfield2010b,Ovaskainen2011,Szewczyk2018,Caradima2019}.

Several quantities were calculated in each sample from the posterior. Probability of presence was calculated as $1 - e^{-\Lambda_{(j,k)s}}$ at the regional scale \citep{Hefley2016}, and $1 - pr(0 | \lambda_{is}, \theta)$ at the local scale. Predicted richness was calculated as the sum of species-level Bernoulli draws based on the probability of presence. Shannon's H was calculated using $\lambda_{i,1:S}$ and $\Lambda_{(j,k),1:S}$. Total predicted intensity was calculated as $\sum_{s=1}^{S}\Lambda_{(j,k)s}$ and $\sum_{s=1}^{S}\lambda_{is}$. Posterior predictions of local communities were for only structured abundance plots, while regional predictions were for the 44 structured sites and the entire gridded landscape for Vaud (Fig. \ref{fig:VD_map}). 



\subsection{Variable selection \& model fitting}
To evaluate the effect of including the presence-only data, \textbf{W}, we compared a version of the model with and without \textbf{W}. We refer to the model that uses both \textbf{W} and \textbf{Y} as the \emph{Joint} model, and the model using only \textbf{Y} as the \emph{Structured} model. The \emph{Structured} model therefore did not include Eq. \ref{eq:W_Mn}, but nevertheless included species only detected in \textbf{W}. We performed variable selection separately on the two models.

Because comparing all possible models was not logistically feasible, variable selection was performed using a 4-fold cross-validation with a step-wise forward search. The 44 sites in \textbf{Y} were divided into 4 subsets using the R package \emph{caret 6.0.86}. The models were parameterized using 3 of the subsets and predictive ability assessed using the 4$^{th}$, with all subsets predicted in turn. For each successive level of model complexity, the expected log pointwise predictive density was calculated for $I_{oos}$ out-of-sample plots as $\sum_{i=1}^{I_{oos}}\sum_{s=1}^S log(P(Y_{is} | \lambda_{is}, \theta))$ and compared using the R package \emph{loo 2.3.1}. Variables were added sequentially, and the variable sets for the \emph{Joint} and \emph{Structured} models with the best out-of-sample predictive ability were considered optimal. Using the optimal variable sets, the models were fit using all observations, with intensities then predicted across the full landscape, excluding cells with covariates more than three standard deviations beyond the values available for fitting.

We assessed model performance of the optimal models based on deviance using \textbf{Y} such that $d = \sum_{i=1}^{I}\sum_{s=1}^S log(P(Y_{is} | \lambda_{is}, \theta))$ in each sample from the posterior, with the explanatory power of each proposed model as $D^2 = \frac{d_{null} - d_{proposed}}{d_{null}}$ \citep{Caradima2019,Guisan2000}. We used an intercept-only model fit with \textbf{Y} (i.e., \emph{Structured} model) as the null model. We calculated overall $D^2$ using all abundances, species-specific $D^2_s$ to compare performance across species, and plot-specific $D^2_i$ to compare performance across elevations. We calculated each metrics for the fitted full models, and during the 4-fold cross validation to assess improvement in predictions of out-of-sample plots.

All models were written in Stan \citep{Carpenter2017} and run in \emph{CmdStan 2.23.0}. Prior distributions (Table \ref{table:priors}) were lightly informative to constrain the sampling algorithm to plausible ranges \citep{Carpenter2017,Lemoine2019}. During variable selection, we ran 3 chains for each model with 2,500 iterations per chain, with 2,000 iterations as warm-up. For the optimal models, we ran 12 chains with 2,250 iterations each, with 2,000 iterations as warm-up. Model behavior and convergence was assessed by confirming that all $\hat{R}$ values were $\leq$ 1.1, visually inspecting a selection of trace plots including hyperparameters and scale parameters, and confirming that no divergent transitions occurred during the sampling. Highest Posterior Density Intervals (HPDIs) were calculated for all parameters of interest using the R package \emph{HDInterval 0.2.2}. Code for the full model is available in Code A.1 and on GitHub (versioned DOI upon acceptance).


\subsection{Community analyses}
We performed several analyses of the predicted communities with the $\lambda$ posterior medians. We assessed similarity in the taxonomically-weighted community structure among plots with a double principal coordinates analysis (DPCoA) using the R packages \emph{ade4 1.7.15} and \emph{vegan 2.5.6} \citep{Dray2015,Pavoine2019}. Using relative abundances, the DPCoA accounts for relatedness among species, and we used 100m elevational bins and regions to categorize communities \emph{post-hoc}. The elevational distribution of each species was also characterized by the observed elevational range, and by the proportion of total intensity in plateau ($<$1000m) or montane ($\geq$1000m) environments.

With the $\lambda$ posterior medians, we calculated the $\beta$-diversity among plots within each site using the R package \emph{betapart 1.5.1}. The overall $\beta$-diversity at each site was partitioned into the balanced variation and abundance-gradient components, representing changes in relative abundance among species and changes in total abundance, respectively \citep{Baselga2017}. Change across elevation for total $\beta$-diversity and each component was assessed using linear and quadratic regressions compared with AICc using the R package \emph{AICcmodavg 2.3.0}. When $\Delta$AICc was $<$ 4, the linear model was selected as most parsimonious. 




\section{Results}
\label{S:3}
A total of 79 species were detected across both datasets, with 76 in the presence-only dataset, \textbf{W}, and 51 in the structured abundance dataset, \textbf{Y} (Table \ref{table:species_list}). \textbf{W} contained 6,632 samples collected by citizen scientists and 363 supplemental transect or tree samples, covering 42\% of the 1 km$^2$ grid (1,309 cells; occurrences per cell: range=1–162, median=3). \textbf{Y} contained 1,090 colonies detected across the 44 sites (per-site range=6–58, median=22.5). 

% Model performance & covariates
The \emph{Joint} model, fit with \textbf{W} and \textbf{Y}, consistently outperformed the \emph{Structured} model, fit with only \textbf{Y}, in predicting novel local communities during cross-validation, and was more resistant to over-fitting (Fig. \ref{fig:cross-validation}, Table \ref{table:cross-validation}; optimal models: \emph{Joint} $D^2$=0.079; \emph{Structured} $D^2$=0.050). In the full model, the \emph{Structured} model fit the data somewhat more closely (\emph{Joint} $D^2$=0.19; \emph{Structured} $D^2$=0.21). This same pattern held on average across species (median: \emph{Joint} $D^2_s$=0.16; \emph{Structured} $D^2_s$=0.21) and plots (median: \emph{Joint} $D^2_i$=0.27; \emph{Structured} $D^2_i$=0.28). However, the \emph{Joint} model dramatically improved fit for species that were not detected in \textbf{Y} (median: \emph{Joint} $D^2_{s[Not In Y]}$=0.52; \emph{Structured} $D^2_{s[Not In Y]}$=0.36). There was no elevational pattern in $D^2_i$.

\begin{figure}
	\centering\includegraphics[width=3in]{../../../ms/1_Ecography/1/figs/crossvalidation.png}
	\caption{\label{fig:cross-validation} Out-of-sample $D^2$ during cross-validation with model complexity. The \emph{Joint} model (purple) consistently outperformed the \emph{Structured} model (green) in predicting out-of-sample local communities, and showed less susceptibility to over-fitting.  }
\end{figure}

The optimal variables for both models included growing degree days at a regional scale, and local effects of relative soil temperature and vegetation cover (Fig. \ref{fig:slope_means}a). The optimal \emph{Joint} model also included regional effects of forest cover and road length, while the optimal \emph{Structured} model included local effects of canopy type. For many variables, species showed a variety of responses, resulting in no clear aggregate effect as the 95\% Highest Posterior Density Intervals (HPDIs) included zero (Fig. \ref{fig:slope_means}a, horizontal points and lines; Fig. \ref{fig:b_byParam}). However, the aggregate effect of growing degree days was positive in the \emph{Joint} model, and the aggregate effect of squared growing degree days was negative in both models. For relative soil temperature, vegetation cover, and mixed canopy, nearly all species with non-zero HPDIs showed positive responses (Fig. \ref{fig:b_bars}). The \emph{Joint} model reduced uncertainty in species' responses compared to the \emph{Structured} model across regional covariates, as indicated by narrower 95\% HPDIs (Fig. \ref{fig:slope_means}b). This reduction occurred on average across all species, but was greater for species not found in \textbf{Y}. In contrast, uncertainty in the \emph{Joint} model was somewhat higher for local variables.

\begin{figure}
\centering\includegraphics[width=6in]{../../../ms/1_Ecography/1/figs/slope_means+HDI.png}
\caption{\label{fig:slope_means} Covariate effects in optimal models. (a) Density curves summarise species-level posterior means for responses to variables in the model using both datasets (\emph{Joint}: purple) and the model using only the abundance data (\emph{Structured}: green). Points and lines show posterior means and Highest Posterior Density Intervals (HPDI; thick: 80\%; thin: 95\%) for the aggregate responses. (b) Change in 95\% HPDI widths in species responses between models (negative: less uncertainty in \emph{Joint} model) for species observed in the structured abundance dataset \textbf{Y} (dark blue) or only in the presence-only dataset \textbf{W} (light blue). Points and lines show mean $\pm$ standard error across species. }
\end{figure}

% Elevational patterns
Both models predicted a peak in ant richness just below 1000m at a regional scale, with no clear trend at a local scale (Fig. \ref{fig:el_patterns}a). In contrast, the \emph{Joint} model predicted higher regional Shannon diversity at lower elevations, while the \emph{Structured} model predicted little elevational trend (Fig. \ref{fig:el_patterns}b). At a local scale, the \emph{Structured} model predicted lower Shannon diversity at low elevations, increasing until 1000m, while the \emph{Joint} model predicted little trend. The \emph{Structured} model predicted higher regional colony intensities at lower elevations, in contrast to the increase in intensity above 1000m predicted by the \emph{Joint} model (Fig. \ref{fig:el_patterns}c). Local predicted intensity was similar across models, with little trend. 

\begin{figure}
	\centering\includegraphics[width=6in]{../../../ms/1_Ecography/1/figs/el_patterns.png}
	\caption{\label{fig:el_patterns} Elevational patterns of posterior distributions at regional and local scales for (a) species richness, (b) Shannon H diversity, and (c) total colony intensity. Lines and ribbons are loess lines using the posterior medians and 95\% Highest Posterior Density Interval, respectively, for the model parameterized with both datasets (\emph{Joint}: purple) and the model parameterized with only the structured abundance data (\emph{Structured}: green). }
\end{figure}


Local ant communities showed strong separation across elevations. The DPCoA showed two distinct clusters representing plateau ($<$1000m) and montane ($\geq$1000m) plots, with results qualitatively similar for both models (\emph{Joint}: Fig. \ref{fig:DPCoA_assemblages}, \emph{Structured}: \ref{fig:dpcoa_struc}). No clear pattern was seen at lower elevations, with broad, partially overlapping clusters from 300–700m and 700–1000m (Fig. \ref{fig:DPCoA_assemblages}a). Above 1000m, communities showed a gradient, with elevation increasing down the y-axis. Plateau communities formed a notably broad cluster (Fig. \ref{fig:DPCoA_assemblages}b), indicating greater variation among local ant communities. In contrast, montane communities clustered more tightly, with the western Jura mountain plots appearing largely as a subset of those in the taller, eastern Alps. Correspondingly, many species were predicted to occur predominantly in plateau or in montane zones (Fig. \ref{fig:lam_zones}), though 41 species (52\%) exhibited elevational ranges that spanned both (Table \ref{table:species_list}).

\begin{figure}
	\centering\includegraphics[width=\textwidth]{../../../ms/1_Ecography/1/figs/DPCoA_assemblage.png}
	\caption{\label{fig:DPCoA_assemblages} Elevational assemblages in the ant communities. Double principle coordinate analysis (DPCoA) of local communities predicted by the \emph{Joint} model showed clustering by (a) elevational bin and (b) region. The central plateau extends to $\sim$1000m, with the Jura and Alps rising steeply in the west and east, respectively. (c) Genus composition in the presence-only and structured abundance datasets each showed differentiation across plateau and montane environments. Only genera that constitute $\geq 1\%$ of at least one subset are shown, with others indicated as 'Other'.}
\end{figure}

Both datasets showed strong differences in the genus composition between the plateau and montane samples (Fig. \ref{fig:DPCoA_assemblages}c). The plateau ($<$1000m) was dominated by \emph{Lasius spp.}, with high representation of \emph{Formica spp.} in the montane zone ($\geq$1000m). The structured abundance dataset \textbf{Y} also showed high relative abundance of \emph{Myrmica spp.}, particularly in montane environments. Compared to \textbf{Y}, the presence-only dataset \textbf{W} under-represented many species (95\% HPDIs $<$ 0; \emph{Joint}: 23 species = 29\%; Fig. \ref{fig:D}), including nearly every species in the genus \emph{Myrmica}. In contrast, only the anthropophilic pavement ant \emph{Tetramorium immigrans} was clearly over-represented (95\% HPDIs $>$ 0; \emph{Joint}: 1 species = 1\%; Fig. \ref{fig:D}). 

Patterns of $\beta$-diversity among plots within structured sites did not dramatically differ between models (Fig. \ref{fig:beta_div}). Overall $\beta$-diversity showed weak elevational patterns (\emph{Joint}: p=0.03, R$^2$=0.10, $\Delta$AICc=0.7; \emph{Structured}: p=0.06, R$^2$=0.06, $\Delta$AICc=3.1), with relatively high variability among local communities on average regardless of elevation. At low elevations, the \emph{balanced variation} and \emph{abundance gradient} components were comparable, indicating differences in total intensity and in turnover of species identities among plots (\emph{Joint}: p$<$0.001, R$^2$=0.70, $\Delta$AICc=5.4; \emph{Structured}: p$<$0.001, R$^2$=0.48, $\Delta$AICc=0.31). At higher elevations, the \emph{abundance gradient} component dominated; plots differed in total intensity rather than in species identities (\emph{Joint}: p$<$0.001, R$^2$=0.72, $\Delta$AICc=10.6; \emph{Structured}: p$<$0.001, R$^2$=0.44, $\Delta$AICc=2.4). 

\begin{figure}
\centering\includegraphics[width=3in]{../../../ms/1_Ecography/1/figs/beta_diversity.png}
\caption{\label{fig:beta_div} Multi-site $\beta$-diversity and its components across elevation. Posterior medians were used to calculate multi-site $\beta$-diversity and its components within each 1 km$^2$ site, describing variation among plots. \emph{Balanced variation} quantifies changes in relative abundance among species, while \emph{abundance gradient} denotes changes in total abundance.}
\end{figure}


 




\section{Discussion}
\label{S:4}

% Summary
% shorten)
The \emph{Joint} model outperformed the \emph{Structured} model in predicting out-of-sample communities and was less susceptible to over-fitting. The information provided by the presence-only dataset at a regional scale produced more accurate, robust predictions at a local scale. Further, the \emph{Joint} model reduced uncertainty in species-level regional responses, most dramatically for species that were not detected in the structured abundance dataset. In the \emph{Joint} model, regional ant richness was predicted to peak at the base of the mountains, declining in both lower and higher elevations. In contrast, Shannon diversity was predicted to decline somewhat from low to high elevations. Patterns were best predicted by growing degree days, forest cover, and road length at a regional scale, and soil temperature, understory vegetation cover, and canopy cover at a local scale. Local ant communities showed plateau and montane clusters, with distinct changes in the generic composition between zones. At high elevations, the variation among local communities was dominated by changes in total abundance, while at low elevations it was more equally determined by variation in both relative and total abundance.

% Patterns
Ant richness frequently peaks at intermediate elevations, as observed here and in other regions of the Alps \citep{Hellrigl2003,Glaser2006,SzewczykMcCain2016}. While average local plot abundance, richness, and diversity remain largely stable across the gradient, the regional scale decrease in richness and increase in colony intensity at high elevations implies density compensation \citep{LongColw2011}. However, there was great uncertainty in predicted regional intensity at high elevations, especially in the \emph{Joint} model. Several factors likely contribute. The presence-only dataset included more observations of mound-building montane species that can occur at extremely high densities, such as \emph{Formica pressilabris} \citep{Seifert2018}. Additionally, intensity magnitudes are informed by the local plots, which only extended to 2222 m, while the regional intensities were predicted as high as 2912 m. These elevations contain novel conditions beyond those that exist at the structured abundance plots, and so the fitted relationships are extrapolated. For example, several mound-building montane species of \emph{Formica} showed strong negative responses to roads (Fig. \ref{fig:b_byParam}), which are sparse at high elevations. The large uncertainty in the \emph{Joint} model predictions thus reflect the interplay of novel physical environments, limitations of the local abundance data, and species' ecology \citep{Harris2018,Guillera-Arroita2019}. 

% Supported drivers
There was strong support for both regional and local temperatures in determining the structure of the ant community, consistent with much past research \citep{SandersEtAl2007,Penick2017,Bishop2016,SzewczykMcCain2016,Szewczyk2018}. However, thermal preferences were highly variable between species, including among congeners (Fig. \ref{fig:b_byParam}). Responses to canopy, forest cover, and road length were similarly variable, indicating differentiation of broad habitat type preferences among species. In contrast, species with clear responses to soil temperature or local vegetation coverage tended to relatively warm and relatively productive microhabitats (Fig. \ref{fig:b_bars}). Rather than limiting ant abundance by impeding worker activity or reducing temperatures \citep{LassauHochuli2004,Queiroz2016}, the Swiss ant fauna most affected by local productivity preferred denser vegetation \citep{Andersen1986,KaspariEtAl2000}. Nevertheless, only select taxa showed positive relationships, with no effect on the predicted intensity of most species. Specifically, many \emph{Myrmica} species, \emph{Solenopsis fugax}, and two \emph{Lasius} species (\emph{L. niger}, \emph{L. flavus}) were all more likely to occur in more productive microhabitats. Similarly, a relatively small number of species showed strong positive or negative effects of canopy cover. 

% Assemblages
Plateau ($<$ 1000 m) and montane ($>$ 1000 m) local communities were quite distinguishable based on species composition, and tended to form clear clusters (Fig. \ref{fig:DPCoA_assemblages}). However, the communities do not appear to be strongly Clementsian; 52\% of species were detected in both the plateau and the montane, with only 33\% restricted to the plateau and 15\% exclusively montane, and the observed and modelled elevational ranges show many independently distributed species rather than cohesive plateau and montane assemblages (Table \ref{table:species_list}, Fig. \ref{fig:lam_zones}). Local communities also showed more variation in species composition at lower elevations, with greater turnover in species identities among plots. While total $\beta$-diversity was comparable at high elevations, the difference among plots was driven by changes in the total number of expected colonies rather than by which species were likely to occur.

% Limitations, possibilities for extensions
The model described here, like any, required assumptions. The predicted intensity at 1 km$^2$ assumes that the plots within each structured abundance site are representative of the larger area. Spatially discrete models require summarising the variation within each spatial unit, and consequently local variation is not accounted for \citep{Calabrese2014,Frishkoff2019,AminiTehrani2020}. While the plots are reasonably representative within the 44 structured sites (Fig. \ref{fig:VD_map}), extrapolation to the larger grid may fail to account for more extreme heterogeneity in suitability, such that the modelled regional intensity and richness may be better interpreted as an upper bound \citep{Guillera-Arroita2019,Altwegg2019,Johnston2020}. In fact, past work suggests that an underestimation of heterogeneity should be expected, both in individual species' distributions and in the overall patterns of richness, abundance, and diversity, resulting from error in the estimation of species' responses, error in the included predictors, or predictors that were not included \citep{Calabrese2014,Biber2019,Guillera-Arroita2019}. Though the local latent variable attempted to model some combination of missing predictors or biotic interactions, an unfeasibly large number of latent variables may be necessary to accurately capture local dynamics \citep{Ovaskainen2016}. 

% Hierarchical shrinkage and rare species
A key attribute of jointly estimated models is that information can be shared across species. This technique, known as hierarchical shrinkage, leads to improved predictions, particularly for species with sparse data \citep{Fithian2015}. Here, species with little or no data were assumed to show preferences similar to congeners. Thus, in the \emph{Structured} model, species undetected in \textbf{Y} were predicted  to be rare, but to show similar relative responses as other species within the same genus. For example, taxa that do not nest in the soil (e.g., \emph{Camponotus spp.}) were very unlikely to be detected in the structured soil plots (Table \ref{table:species_list}). Consequently, the \emph{Structured} model  predicted that \emph{Camponotus} species were generally rare with a relative pattern informed by the aggregate ($\beta$) responses as well as the limited detections of \emph{C. ligniperdus} (Fig. \ref{fig:gen_map_Structured}, \ref{fig:b_byParam}). In contrast, the \emph{Joint} model predictions were rather different, because the preferences of these species were further informed by \textbf{W} (Fig. \ref{fig:gen_map_Joint}, \ref{fig:b_byParam}). Nevertheless, species with few total observations were more similar to the genus-level average. Thus, given a paucity of species-level data, hierarchical shrinkage provides a 'best guess' based on other species, with correspondingly large uncertainty (Fig. \ref{fig:slope_means}b, Fig. \ref{fig:b_byParam}).

% Bias in citizen science data
The presence-only dataset \textbf{W} showed clear spatial and taxonomic bias. These are common issues in citizen science projects \citep{Ward2014,Theobald2015,Johnston2020,Robinson2020}, and the model structure explicitly incorporated these biases \citep{Isaac2014}. Though coverage was expansive through the region (Fig. \ref{fig:VD_map}), many grid cells were represented by a single sample, with a few cells containing more than one hundred. Similarly, the taxonomic composition of \textbf{W} was not fully aligned with \textbf{Y}, which more accurately quantified colony density (Fig. \ref{fig:D}, Table \ref{table:species_list}). While some anthropophilic species were likely over-represented, \textbf{W} showed a stronger tendency toward under-representation. Many of the under-represented taxa have somewhat unobstrusive lifestyles or tend to live in less trafficked habitats \citep{Seifert2018}, nesting and foraging in denser vegetation (e.g., some \emph{Myrmica} species), under the soil (e.g., some \emph{Lasius} species), or in twigs (e.g., \emph{Temnothorax} species). Further, these taxa are not so rare or morphologically unique that they would be specifically sought after by contributing experts or enthusiasts. 

% Contribution of citizen science data
Despite these biases, the presence-only dataset also made clear contributions. The citizen science efforts greatly expanded the taxonomic breadth of the survey, including 28 species (35\% of total) and 7 genera (33\% of total) that were not detected in \textbf{Y} (Table \ref{table:species_list}). Opportunistic datasets often better capture uncommon species \citep{Henckel2020}. Correspondingly, the taxa detected only in \textbf{W} tended to be rare, including 9 singletons and 4 doubletons. The integration of the two datasets consequently allowed for more precise estimation of species responses (Fig. \ref{fig:slope_means}b), particularly for species with few observations. As a result, the presences, incorporated into the model only at a regional scale, improved out-of-sample predictions at a local scale (Fig. \ref{fig:cross-validation}). The presence-only dataset also rendered the \emph{Joint} model less susceptible to over-fitting. In contrast, the \emph{Structured} model declined in predictive performance at rather modest levels of complexity, performing worse than the intercept-only model when more than 9 covariates were included, while the \emph{Joint} model showed only limited declines in performance. 

% Conclusion with a focus on the value of contribution of the citizen scientists
The efforts of citizen scientists in the canton of Vaud thus contributed valuable information not captured by the concurrent structured samples. The broad spatial coverage captured a similarly broad array of species, including rarer species that were unlikely to be detected in a structured survey focused on colony density. The \emph{Joint} model accounted for spatial and taxonomic bias, as well as geo-locational imprecision, to improve the estimation of species' responses to environmental variables as well as the predicted distribution of ants across the region. By leveraging the strengths of each collection effort, the hierarchical model described here provides a framework for leveraging the efforts of citizen scientists to improve our understanding of the distribution of biodiversity.




\section{Acknowledgments}
We offer thanks to all of the citizen scientists who contributed to Opération Fourmis, as well as the researchers, staff, and expert taxonomists associated with the project. We further thank all of the volunteers and technicians who dedicated hours to collecting the structured samples.

\newpage
\section{Bibliography}
\bibliography{../../../ms/opfo_diversity}



\newpage
\section{Appendix Descriptions}

\subsection{Appendix A. Supplemental methods.}

\textbf{Table \ref{table:landcover}.} Land cover types and canopy classification

\textbf{Table \ref{table:priors}.} Prior distributions

\textbf{Code A.1.} Stan code for full model



\subsection{Appendix B. Supplemental results.}

\textbf{Table \ref{table:species_list}.} Species list with observed elevational range and counts

\textbf{Table \ref{table:cross-validation}.} Variable selection results

\textbf{Figure \ref{fig:b_byParam}.} Species-level responses (points + HPDIs)

\textbf{Figure \ref{fig:b_bars}.} Species-level directional response summaries (bar proportions)

\textbf{Figure \ref{fig:lam_zones}.} Plateau vs montane species' intensities

\textbf{Figure \ref{fig:dpcoa_struc}.} DPCoA based on \emph{Structured} model predictions

\textbf{Figure \ref{fig:D}.} Taxonomic bias

\textbf{Figure \ref{fig:sf_map_Joint}.} Maps of predicted richness by subfamily (\emph{Joint})

\textbf{Figure \ref{fig:sf_map_Structured}.} Maps of predicted richness by subfamily (\emph{Structured})

\textbf{Figure \ref{fig:gen_map_Joint}.} Maps of predicted richness by genus (\emph{Joint})

\textbf{Figure \ref{fig:gen_map_Structured}.} Maps of predicted richness by genus (\emph{Structured})



\newpage
\appendix

\section{Supplemental Methods}

\begin{table}[ht]
	\centering
	\begin{tabular}{ l l c c c}
		\hline
		\textbf{Land cover type} & \textbf{Canopy} & \textbf{Plots} & \textbf{Colonies} & \textbf{Species} \\
		\hline
		Mixed forest & Closed & 82 (7.7\%) & 68 (6.2\%) & 14 (27.5\%) \\
		Deciduous forest & Closed & 97 (9.2\%) & 68 (6.2\%) & 15 (29.4\%) \\
		Conifer forest & Closed & 99 (9.3\%) & 55 (5.0\%) & 13 (25.5\%) \\
		Alluvial zone & Mixed & 3 (0.3\%) & 4 (0.4\%) & 2 (3.9\%) \\
		Orchards \& vineyards & Mixed & 16 (1.5\%) & 29 (2.7\%) & 6 (11.8\%) \\
		Roadside & Mixed & 46 (4.3\%) & 67 (6.1\%) & 16 (31.4\%) \\
		Transition & Mixed & 71 (6.7\%) & 70 (6.4\%) & 23 (45.1\%) \\
		Quarry & Open & 2 (0.2\%) & 1 (0.1\%) & 1 (2.0\%) \\
		Gravel & Open & 4 (0.4\%) & 4 (0.4\%) & 2 (3.9\%) \\
		Marsh & Open & 6 (0.6\%) & 11 (1.0\%) & 5 (9.8\%) \\
		Wetland & Open & 7 (0.7\%) & 3 (0.3\%) & 3 (5.9\%) \\
		Scree & Open & 23 (2.2\%) & 8 (0.7\%) & 6 (11.8\%) \\
		Dry meadow & Open & 38 (3.6\%) & 63 (5.8\%) & 19 (37.3\%) \\
		Built zone & Open & 52 (4.9\%) & 57 (5.2\%) & 13 (25.5\%) \\
		Other open & Open & 513 (48.4\%) & 582 (53.4\%) & 35 (68.6\%) \\
	\end{tabular}
	\caption{\label{table:landcover} Land cover types and canopy classifications with the number of plots, colonies, and species detected in the structured abundance dataset. The percent of the column total is shown in parentheses. }
\end{table}



\newpage

\begin{table}[ht]
	\centering
	\begin{tabular}{c}
		\hline
		$\alpha_x \sim Normal(\mu=0, \sigma=1)$ \\
		$\beta_x \sim Normal(\mu=0, \sigma=1)$ \\
		$\beta_0 \sim Normal(\mu=-4, \sigma=2)$ \\
		$\sigma_s \sim Normal(\mu=0, \sigma=2) \ T(0,\infty)$\\
		$\sigma_g \sim Normal(\mu=0, \sigma=2) \ T(0,\infty)$\\
		$\Omega_g \sim LKJ(\eta=2)$ \\
		$\zeta \sim Normal(\mu=0, \sigma=1)$\\
		$\gamma \sim Normal(\mu=0, \sigma=1) \ T(-1,1)$\\
		$\theta \sim Uniform(0,1)$ \\
		$D \sim Normal(\mu=1, \sigma=1) \ T(0,\infty)$ \\
		\hline
	\end{tabular}
	\caption{\label{table:priors} Prior distributions. Prior distributions were lightly informative, constraining values to plausible ranges to improve sampling. The correlation matrices for the genus-level slopes used a Cholesky decomposition and followed an LKJ prior as recommended for better model performance in Stan \citep{Carpenter2017,Caradima2019}.}
\end{table}



\newpage

\lstset{basicstyle=\scriptsize}
\lstinputlisting[title=\label{code:full_model} Code A.1: Stan code for full model., language=Stan]{../../../code/mods/full_model_pub.stan}





\newpage

\section{Supplemental Results}

\setlength{\LTcapwidth}{\textwidth}
\begingroup\small
\begin{longtable}{ l c c c c}
	\caption{\label{table:species_list} Species list with observed elevational minimum and maximum (m), and counts in the presence-only dataset \textbf{W} and structured abundance dataset \textbf{Y}. } \\
	\hline
	\textbf{Species} & \textbf{Minimum} & \textbf{Maximum} & \textbf{W} & \textbf{Y}  \\
	\hline
	\endfirsthead
	\caption[]{(continued)}\\
	\hline
	\textbf{Species} & \textbf{Minimum} & \textbf{Maximum} & \textbf{W} & \textbf{Y}  \\
	\hline
	\endhead
	\emph{Aphaenogaster subterranea} & 395 & 710 & 39 (0.6\%) & 2 (0.2\%) \\
	\emph{Camponotus aethiops} & 430 & 430 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Camponotus fallax} & 369 & 459 & 16 (0.2\%) & 0 (0.0\%) \\
	\emph{Camponotus herculeanus} & 665 & 1839 & 38 (0.5\%) & 0 (0.0\%) \\
	\emph{Camponotus ligniperda} & 381 & 1497 & 123 (1.8\%) & 3 (0.3\%) \\
	\emph{Camponotus piceus} & 430 & 1323 & 13 (0.2\%) & 0 (0.0\%) \\
	\emph{Colobopsis truncata} & 369 & 803 & 24 (0.3\%) & 1 (0.1\%) \\
	\emph{Crematogaster scutellaris} & 415 & 415 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Dolichoderus quadripunctatus} & 383 & 746 & 16 (0.2\%) & 0 (0.0\%) \\
	\emph{Formica cinerea} & 365 & 1085 & 3 (0.0\%) & 1 (0.1\%) \\
	\emph{Formica cunicularia} & 369 & 1874 & 419 (6.0\%) & 10 (0.9\%) \\
	\emph{Formica exsecta} & 918 & 1738 & 13 (0.2\%) & 0 (0.0\%) \\
	\emph{Formica fusca} & 414 & 1984 & 132 (1.9\%) & 32 (2.9\%) \\
	\emph{Formica fuscocinerea} & 950 & 950 & 2 (0.0\%) & 0 (0.0\%) \\
	\emph{Formica lemani} & 607 & 2332 & 144 (2.1\%) & 111 (10.2\%) \\
	\emph{Formica lugubris/paralugubris} & 489 & 2151 & 514 (7.3\%) & 26 (2.4\%) \\
	\emph{Formica picea} & 1876 & 1876 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Formica polyctena} & 396 & 1612 & 179 (2.6\%) & 1 (0.1\%) \\
	\emph{Formica pratensis} & 376 & 1984 & 91 (1.3\%) & 5 (0.5\%) \\
	\emph{Formica pressilabris} & 1113 & 2285 & 239 (3.4\%) & 24 (2.2\%) \\
	\emph{Formica rufa} & 398 & 1985 & 258 (3.7\%) & 4 (0.4\%) \\
	\emph{Formica rufibarbis} & 369 & 1523 & 185 (2.6\%) & 5 (0.5\%) \\
	\emph{Formica sanguinea} & 606 & 1695 & 83 (1.2\%) & 2 (0.2\%) \\
	\emph{Formica selysi} & 934 & 1792 & 2 (0.0\%) & 0 (0.0\%) \\
	\emph{Formica truncorum} & 1018 & 1407 & 6 (0.1\%) & 0 (0.0\%) \\
	\emph{Formicoxenus nitidulus} & 1143 & 1826 & 8 (0.1\%) & 1 (0.1\%) \\
	\emph{Harpagoxenus sublaevis} & 1428 & 1428 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Lasius alienus} & 414 & 769 & 9 (0.1\%) & 1 (0.1\%) \\
	\emph{Lasius brunneus} & 379 & 1266 & 78 (1.1\%) & 4 (0.4\%) \\
	\emph{Lasius emarginatus} & 369 & 1695 & 486 (6.9\%) & 2 (0.2\%) \\
	\emph{Lasius flavus} & 365 & 1783 & 368 (5.3\%) & 134 (12.3\%) \\
	\emph{Lasius fuliginosus} & 372 & 1479 & 182 (2.6\%) & 6 (0.6\%) \\
	\emph{Lasius mixtus} & 502 & 1319 & 5 (0.1\%) & 1 (0.1\%) \\
	\emph{Lasius myops} & 394 & 639 & 4 (0.1\%) & 0 (0.0\%) \\
	\emph{Lasius niger} & 367 & 1803 & 1381 (19.7\%) & 186 (17.1\%) \\
	\emph{Lasius paralienus} & 430 & 848 & 33 (0.5\%) & 8 (0.7\%) \\
	\emph{Lasius platythorax} & 368 & 1440 & 192 (2.7\%) & 11 (1.0\%) \\
	\emph{Lasius psammophilus} & 432 & 432 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Lasius umbratus} & 428 & 428 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Leptothorax acervorum} & 473 & 2043 & 30 (0.4\%) & 4 (0.4\%) \\
	\emph{Leptothorax gredleri} & 1319 & 1337 & 3 (0.0\%) & 0 (0.0\%) \\
	\emph{Leptothorax muscorum} & 1494 & 1494 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Manica rubida} & 726 & 2183 & 54 (0.8\%) & 18 (1.7\%) \\
	\emph{Myrmecina graminicola} & 373 & 747 & 25 (0.4\%) & 2 (0.2\%) \\
	\emph{Myrmica curvithorax} & 495 & 694 & 1 (0.0\%) & 2 (0.2\%) \\
	\emph{Myrmica gallienii} & 429 & 429 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Myrmica lobicornis} & 1227 & 1759 & 1 (0.0\%) & 6 (0.6\%) \\
	\emph{Myrmica lobulicornis} & 1107 & 1929 & 6 (0.1\%) & 15 (1.4\%) \\
	\emph{Myrmica lonae} & 1210 & 1490 & 1 (0.0\%) & 1 (0.1\%) \\
	\emph{Myrmica rubra} & 365 & 1552 & 92 (1.3\%) & 23 (2.1\%) \\
	\emph{Myrmica ruginodis} & 371 & 2014 & 276 (3.9\%) & 116 (10.6\%) \\
	\emph{Myrmica rugulosa} & 502 & 502 & 0 (0.0\%) & 1 (0.1\%) \\
	\emph{Myrmica sabuleti} & 365 & 1876 & 118 (1.7\%) & 39 (3.6\%) \\
	\emph{Myrmica scabrinodis} & 399 & 2030 & 34 (0.5\%) & 82 (7.5\%) \\
	\emph{Myrmica schencki} & 401 & 2103 & 11 (0.2\%) & 5 (0.5\%) \\
	\emph{Myrmica specioides} & 369 & 1317 & 49 (0.7\%) & 7 (0.6\%) \\
	\emph{Myrmica sulcinodis} & 1687 & 1821 & 0 (0.0\%) & 2 (0.2\%) \\
	\emph{Myrmica vandeli} & 1041 & 1442 & 0 (0.0\%) & 3 (0.3\%) \\
	\emph{Plagiolepis pallidula} & 424 & 591 & 40 (0.6\%) & 0 (0.0\%) \\
	\emph{Polyergus rufescens} & 384 & 763 & 4 (0.1\%) & 0 (0.0\%) \\
	\emph{Ponera coarctata} & 400 & 623 & 12 (0.2\%) & 0 (0.0\%) \\
	\emph{Ponera testacea} & 430 & 430 & 1 (0.0\%) & 0 (0.0\%) \\
	\emph{Solenopsis fugax} & 366 & 1511 & 106 (1.5\%) & 34 (3.1\%) \\
	\emph{Stenamma debile} & 384 & 734 & 2 (0.0\%) & 0 (0.0\%) \\
	\emph{Tapinoma darioi} & 369 & 427 & 4 (0.1\%) & 0 (0.0\%) \\
	\emph{Tapinoma erraticum} & 369 & 1444 & 195 (2.8\%) & 8 (0.7\%) \\
	\emph{Tapinoma magnum} & 369 & 1283 & 25 (0.4\%) & 0 (0.0\%) \\
	\emph{Tapinoma subboreale} & 378 & 1087 & 54 (0.8\%) & 3 (0.3\%) \\
	\emph{Temnothorax affinis} & 365 & 912 & 28 (0.4\%) & 1 (0.1\%) \\
	\emph{Temnothorax corticalis} & 434 & 657 & 2 (0.0\%) & 0 (0.0\%) \\
	\emph{Temnothorax interruptus} & 424 & 1184 & 4 (0.1\%) & 0 (0.0\%) \\
	\emph{Temnothorax luteus/parvulus} & 613 & 623 & 2 (0.0\%) & 1 (0.1\%) \\
	\emph{Temnothorax nigriceps} & 723 & 1631 & 17 (0.2\%) & 2 (0.2\%) \\
	\emph{Temnothorax nylanderi} & 371 & 1341 & 71 (1.0\%) & 68 (6.2\%) \\
	\emph{Temnothorax tuberum} & 400 & 1524 & 4 (0.1\%) & 2 (0.2\%) \\
	\emph{Temnothorax unifasciatus} & 369 & 1328 & 49 (0.7\%) & 1 (0.1\%) \\
	\emph{Tetramorium caespitum} & 365 & 1683 & 305 (4.4\%) & 39 (3.6\%) \\
	\emph{Tetramorium immigrans} & 366 & 545 & 22 (0.3\%) & 0 (0.0\%) \\
	\emph{Tetramorium impurum} & 759 & 1838 & 54 (0.8\%) & 24 (2.2\%) \\
\end{longtable}
\endgroup


\newpage

\setlength{\LTcapwidth}{\textwidth}
\begingroup\small
\begin{longtable}{ c c c c c c c}
	\caption{\label{table:cross-validation} Results of the 4-fold cross-validation. During the forward search, covariates were added consecutively, using expected log pointwise density (elpd) to assess performance. Bolded variable names indicate those that were kept for each level of model complexity up to the optimal models. } \\
	\hline
	\textbf{Model} & \textbf{Complexity} & \textbf{Variable} & \textbf{Scale} & \textbf{elpd} & \textbf{$\Delta$elpd}& \textbf{$D^2$}  \\
	\hline
	\endfirsthead
	\caption[]{(continued)}\\
	\hline
	\textbf{Model} & \textbf{Complexity} & \textbf{Variable} & \textbf{Scale} & \textbf{elpd} & \textbf{$\Delta$elpd}& \textbf{$D^2$}  \\
	\hline
	\endhead
	\textbf{\emph{Structured}} & \textbf{0} & \textbf{(intercept)} & \textbf{Regional} & \textbf{-3994.5} & \textbf{-199.1} & \textbf{0} \\
	\textbf{\emph{Structured}} & \textbf{1} & \textbf{Growing deg. days} & \textbf{Regional} & \textbf{-3862.3} & \textbf{-66.9} & \textbf{0.033} \\
	\emph{Structured} & 1 & Canopy Type & Local & -3934.4 & -139.0 & 0.015 \\
	\emph{Structured} & 1 & Edge proportion & Regional & -3938.6 & -143.2 & 0.014 \\
	\emph{Structured} & 1 & Soil temperature & Local & -3942.4 & -147.0 & 0.013 \\
	\emph{Structured} & 1 & Pasture/Crop & Local & -3954.5 & -159.1 & 0.01 \\
	\emph{Structured} & 1 & Agriculture proportion & Regional & -3970.0 & -174.6 & 0.006 \\
	\emph{Structured} & 1 & Road length & Regional & -3973.6 & -178.2 & 0.005 \\
	\emph{Structured} & 1 & Local veg. cover & Local & -3984.9 & -189.4 & 0.002 \\
	\emph{Structured} & 1 & H'(Land cover) & Regional & -4010.6 & -215.2 & -0.004 \\
	\emph{Structured} & 1 & Building perimeter & Regional & -4019.9 & -224.5 & -0.006 \\
	\emph{Structured} & 1 & Annual precipitation & Regional & -4047.8 & -252.4 & -0.013 \\
	\emph{Structured} & 1 & Forest proportion & Regional & -4067.9 & -272.5 & -0.018 \\
	\emph{Structured} & 1 & North aspect & Regional & -4073.3 & -277.9 & -0.02 \\
	\emph{Structured} & 1 & NPP & Regional & -4162.7 & -367.3 & -0.042 \\
	\textbf{\emph{Structured}} & \textbf{2} & \textbf{Soil temperature} & \textbf{Local} & \textbf{-3801.0} & \textbf{-5.6} & \textbf{0.048} \\
	\emph{Structured} & 2 & Canopy Type & Local & -3806.0 & -10.6 & 0.047 \\
	\emph{Structured} & 2 & Local veg. cover & Local & -3847.8 & -52.4 & 0.037 \\
	\emph{Structured} & 2 & Pasture/Crop & Local & -3851.2 & -55.8 & 0.036 \\
	\emph{Structured} & 2 & Building perimeter & Regional & -3885.7 & -90.3 & 0.027 \\
	\emph{Structured} & 2 & Edge proportion & Regional & -3889.4 & -94.0 & 0.026 \\
	\emph{Structured} & 2 & Forest proportion & Regional & -3896.2 & -100.8 & 0.025 \\
	\emph{Structured} & 2 & H'(Land cover) & Regional & -3897.8 & -102.4 & 0.024 \\
	\emph{Structured} & 2 & NPP & Regional & -3903.8 & -108.3 & 0.023 \\
	\emph{Structured} & 2 & Road length & Regional & -3906.4 & -111.0 & 0.022 \\
	\emph{Structured} & 2 & Agriculture proportion & Regional & -3917.9 & -122.5 & 0.019 \\
	\emph{Structured} & 2 & North aspect & Regional & -3925.2 & -129.8 & 0.017 \\
	\emph{Structured} & 2 & Annual precipitation & Regional & -3936.6 & -141.2 & 0.015 \\
	\textbf{\emph{Structured}} & \textbf{3} & \textbf{Canopy Type} & \textbf{Local} & \textbf{-3799.1} & \textbf{-3.7} & \textbf{0.049} \\
	\emph{Structured} & 3 & Local veg. cover & Local & -3805.8 & -10.4 & 0.047 \\
	\emph{Structured} & 3 & Building perimeter & Regional & -3820.5 & -25.0 & 0.044 \\
	\emph{Structured} & 3 & Pasture/Crop & Local & -3821.6 & -26.2 & 0.043 \\
	\emph{Structured} & 3 & Forest proportion & Regional & -3831.0 & -35.5 & 0.041 \\
	\emph{Structured} & 3 & Edge proportion & Regional & -3831.6 & -36.2 & 0.041 \\
	\emph{Structured} & 3 & H'(Land cover) & Regional & -3849.3 & -53.9 & 0.036 \\
	\emph{Structured} & 3 & Road length & Regional & -3853.7 & -58.2 & 0.035 \\
	\emph{Structured} & 3 & NPP & Regional & -3862.0 & -66.6 & 0.033 \\
	\emph{Structured} & 3 & Agriculture proportion & Regional & -3862.0 & -66.6 & 0.033 \\
	\emph{Structured} & 3 & North aspect & Regional & -3873.1 & -77.6 & 0.03 \\
	\emph{Structured} & 3 & Annual precipitation & Regional & -3893.6 & -98.2 & 0.025 \\
	\textbf{\emph{Structured}} & \textbf{4} & \textbf{Local veg. cover} & \textbf{Local} & \textbf{-3795.4} & \textbf{0.0} & \textbf{0.05} \\
	\emph{Structured} & 4 & Pasture/Crop & Local & -3801.2 & -5.8 & 0.048 \\
	\emph{Structured} & 4 & Building perimeter & Regional & -3801.7 & -6.3 & 0.048 \\
	\emph{Structured} & 4 & Edge proportion & Regional & -3821.2 & -25.7 & 0.043 \\
	\emph{Structured} & 4 & Forest proportion & Regional & -3831.3 & -35.9 & 0.041 \\
	\emph{Structured} & 4 & H'(Land cover) & Regional & -3833.4 & -38.0 & 0.04 \\
	\emph{Structured} & 4 & NPP & Regional & -3838.5 & -43.1 & 0.039 \\
	\emph{Structured} & 4 & Road length & Regional & -3843.1 & -47.7 & 0.038 \\
	\emph{Structured} & 4 & North aspect & Regional & -3853.8 & -58.4 & 0.035 \\
	\emph{Structured} & 4 & Agriculture proportion & Regional & -3866.1 & -70.7 & 0.032 \\
	\emph{Structured} & 4 & Annual precipitation & Regional & -3895.9 & -100.5 & 0.025 \\
	﻿\emph{Structured} & 5 & Pasture/Crop & Local & -3818.3 & -22.9 & 0.044 \\
	\emph{Structured} & 5 & Building perimeter & Regional & -3829.8 & -34.4 & 0.041 \\
	\emph{Structured} & 5 & Edge proportion & Regional & -3834.6 & -39.1 & 0.04 \\
	\emph{Structured} & 5 & Forest proportion & Regional & -3835.1 & -39.6 & 0.04 \\
	\emph{Structured} & 5 & Road length & Regional & -3844.3 & -48.8 & 0.038 \\
	\emph{Structured} & 5 & H'(Land cover) & Regional & -3852.2 & -56.8 & 0.036 \\
	\emph{Structured} & 5 & NPP & Regional & -3853.9 & -58.5 & 0.035 \\
	\emph{Structured} & 5 & North aspect & Regional & -3863.6 & -68.2 & 0.033 \\
	\emph{Structured} & 5 & Agriculture proportion & Regional & -3879.3 & -83.8 & 0.029 \\
	\emph{Structured} & 5 & Annual precipitation & Regional & -3904.1 & -108.6 & 0.023 \\
	\emph{Structured} & 6 & H'(Land cover) & Regional & -3835.1 & -39.7 & 0.04 \\
	\emph{Structured} & 6 & Building perimeter & Regional & -3835.8 & -40.4 & 0.04 \\
	\emph{Structured} & 6 & Road length & Regional & -3839.4 & -44.0 & 0.039 \\
	\emph{Structured} & 6 & Edge proportion & Regional & -3843.0 & -47.6 & 0.038 \\
	\emph{Structured} & 6 & Forest proportion & Regional & -3845.2 & -49.8 & 0.037 \\
	\emph{Structured} & 6 & NPP & Regional & -3857.9 & -62.5 & 0.034 \\
	\emph{Structured} & 6 & North aspect & Regional & -3884.7 & -89.3 & 0.027 \\
	\emph{Structured} & 6 & Agriculture proportion & Regional & -3886.6 & -91.2 & 0.027 \\
	\emph{Structured} & 6 & Annual precipitation & Regional & -3904.1 & -108.7 & 0.023 \\
	\emph{Structured} & 7 & Building perimeter & Regional & -3861.0 & -65.6 & 0.033 \\
	\emph{Structured} & 7 & Forest proportion & Regional & -3868.2 & -72.8 & 0.032 \\
	\emph{Structured} & 7 & Edge proportion & Regional & -3873.6 & -78.2 & 0.03 \\
	\emph{Structured} & 7 & Road length & Regional & -3892.1 & -96.7 & 0.026 \\
	\emph{Structured} & 7 & NPP & Regional & -3907.9 & -112.5 & 0.022 \\
	\emph{Structured} & 7 & North aspect & Regional & -3908.7 & -113.3 & 0.021 \\
	\emph{Structured} & 7 & Agriculture proportion & Regional & -3916.6 & -121.2 & 0.02 \\
	\emph{Structured} & 7 & Annual precipitation & Regional & -3926.0 & -130.5 & 0.017 \\
	\emph{Structured} & 8 & Forest proportion & Regional & -3892.0 & -96.6 & 0.026 \\
	\emph{Structured} & 8 & Road length & Regional & -3920.8 & -125.4 & 0.018 \\
	\emph{Structured} & 8 & North aspect & Regional & -3929.2 & -133.7 & 0.016 \\
	\emph{Structured} & 8 & Annual precipitation & Regional & -3931.8 & -136.3 & 0.016 \\
	\emph{Structured} & 8 & NPP & Regional & -3932.4 & -137.0 & 0.016 \\
	\emph{Structured} & 8 & Agriculture proportion & Regional & -3943.6 & -148.1 & 0.013 \\
	\emph{Structured} & 8 & Edge proportion & Regional & -3950.1 & -154.7 & 0.011 \\
	\emph{Structured} & 9 & Road length & Regional & -3954.9 & -159.5 & 0.01 \\
	\emph{Structured} & 9 & Annual precipitation & Regional & -3971.7 & -176.2 & 0.006 \\
	\emph{Structured} & 9 & North aspect & Regional & -3974.2 & -178.8 & 0.005 \\
	\emph{Structured} & 9 & Agriculture proportion & Regional & -3987.0 & -191.6 & 0.002 \\
	\emph{Structured} & 9 & NPP & Regional & -3988.1 & -192.6 & 0.002 \\
	\emph{Structured} & 9 & Edge proportion & Regional & -4007.5 & -212.1 & -0.003 \\
	\emph{Structured} & 10 & Agriculture proportion & Regional & -3977.0 & -181.6 & 0.004 \\
	\emph{Structured} & 10 & NPP & Regional & -4033.4 & -238.0 & -0.01 \\
	\emph{Structured} & 10 & Annual precipitation & Regional & -4040.8 & -245.4 & -0.012 \\
	\emph{Structured} & 10 & North aspect & Regional & -4045.9 & -250.5 & -0.013 \\
	\emph{Structured} & 10 & Edge proportion & Regional & -4137.6 & -342.1 & -0.036 \\
	\emph{Structured} & 11 & Annual precipitation & Regional & -4087.9 & -292.5 & -0.023 \\
	\emph{Structured} & 11 & North aspect & Regional & -4133.4 & -338.0 & -0.035 \\
	\emph{Structured} & 11 & NPP & Regional & -4146.4 & -351.0 & -0.038 \\
	\emph{Structured} & 11 & Edge proportion & Regional & -4234.1 & -438.7 & -0.06 \\
	\emph{Structured} & 12 & North aspect & Regional & -4180.0 & -384.5 & -0.046 \\
	\emph{Structured} & 12 & Edge proportion & Regional & -4214.5 & -419.0 & -0.055 \\
	\emph{Structured} & 12 & NPP & Regional & -4274.1 & -478.6 & -0.07 \\
	\emph{Structured} & 13 & NPP & Regional & -4271.4 & -476.0 & -0.069 \\
	\emph{Structured} & 13 & Edge proportion & Regional & -4282.3 & -486.9 & -0.072 \\
	\emph{Structured} & 14 & Edge proportion & Regional & -4425.6 & -630.2 & -0.108 \\
	\textbf{\emph{Joint}} & \textbf{0} & \textbf{(intercept)} & \textbf{Regional} & \textbf{-3988.0} & \textbf{-308.5} & \textbf{0.002} \\
	\textbf{\emph{Joint}} & \textbf{1} & \textbf{Growing deg. days} & \textbf{Regional} & \textbf{-3764.9} & \textbf{-85.4} & \textbf{0.057} \\
	\emph{Joint} & 1 & Agriculture proportion & Regional & -3839.3 & -159.8 & 0.039 \\
	\emph{Joint} & 1 & Road length & Regional & -3879.8 & -200.3 & 0.029 \\
	\emph{Joint} & 1 & Edge proportion & Regional & -3909.5 & -230.0 & 0.021 \\
	\emph{Joint} & 1 & Soil temperature & Local & -3937.5 & -258.0 & 0.014 \\
	\emph{Joint} & 1 & Pasture/Crop & Local & -3957.0 & -277.5 & 0.009 \\
	\emph{Joint} & 1 & Canopy Type & Local & -3961.3 & -281.8 & 0.008 \\
	\emph{Joint} & 1 & Local veg. cover & Local & -3981.2 & -301.7 & 0.003 \\
	\emph{Joint} & 1 & Building perimeter & Regional & -3981.3 & -301.8 & 0.003 \\
	\emph{Joint} & 1 & H'(Land cover) & Regional & -3983.2 & -303.7 & 0.003 \\
	\emph{Joint} & 1 & North aspect & Regional & -3994.6 & -315.1 & 0 \\
	\emph{Joint} & 1 & Forest proportion & Regional & -4015.7 & -336.2 & -0.005 \\
	\emph{Joint} & 1 & NPP & Regional & -4021.4 & -341.9 & -0.007 \\
	\emph{Joint} & 1 & Annual precipitation & Regional & -4120.8 & -441.3 & -0.032 \\
	\textbf{\emph{Joint}} & \textbf{2} & \textbf{Soil temperature} & \textbf{Local} & \textbf{-3707.5} & \textbf{-28.0} & \textbf{0.072} \\
	\emph{Joint} & 2 & Canopy Type & Local & -3740.6 & -61.1 & 0.064 \\
	\emph{Joint} & 2 & Local veg. cover & Local & -3747.4 & -67.9 & 0.062 \\
	\emph{Joint} & 2 & Forest proportion & Regional & -3756.0 & -76.5 & 0.06 \\
	\emph{Joint} & 2 & H'(Land cover) & Regional & -3760.9 & -81.4 & 0.058 \\
	\emph{Joint} & 2 & Agriculture proportion & Regional & -3762.4 & -82.9 & 0.058 \\
	\emph{Joint} & 2 & Road length & Regional & -3764.3 & -84.8 & 0.058 \\
	\emph{Joint} & 2 & Pasture/Crop & Local & -3765.6 & -86.1 & 0.057 \\
	\emph{Joint} & 2 & NPP & Regional & -3772.8 & -93.3 & 0.055 \\
	\emph{Joint} & 2 & Edge proportion & Regional & -3781.9 & -102.4 & 0.053 \\
	\emph{Joint} & 2 & North aspect & Regional & -3784.4 & -104.9 & 0.053 \\
	\emph{Joint} & 2 & Building perimeter & Regional & -3785.9 & -106.4 & 0.052 \\
	\emph{Joint} & 2 & Annual precipitation & Regional & -3804.3 & -124.8 & 0.048 \\
	\textbf{\emph{Joint}} & \textbf{3} & \textbf{Forest proportion} & \textbf{Regional} & \textbf{-3695.4} & \textbf{-15.9} & \textbf{0.075} \\
	\emph{Joint} & 3 & Agriculture proportion & Regional & -3703.8 & -24.3 & 0.073 \\
	\emph{Joint} & 3 & Local veg. cover & Local & -3709.1 & -29.6 & 0.071 \\
	\emph{Joint} & 3 & H'(Land cover) & Regional & -3712.8 & -33.3 & 0.071 \\
	\emph{Joint} & 3 & NPP & Regional & -3714.7 & -35.2 & 0.07 \\
	\emph{Joint} & 3 & Road length & Regional & -3715.6 & -36.1 & 0.07 \\
	\emph{Joint} & 3 & Building perimeter & Regional & -3724.2 & -44.7 & 0.068 \\
	\emph{Joint} & 3 & Canopy Type & Local & -3725.8 & -46.3 & 0.067 \\
	\emph{Joint} & 3 & Pasture/Crop & Local & -3726.1 & -46.6 & 0.067 \\
	\emph{Joint} & 3 & North aspect & Regional & -3730.4 & -50.9 & 0.066 \\
	\emph{Joint} & 3 & Edge proportion & Regional & -3731.7 & -52.2 & 0.066 \\
	\emph{Joint} & 3 & Annual precipitation & Regional & -3759.9 & -80.4 & 0.059 \\
	\textbf{\emph{Joint}} & \textbf{4} & \textbf{Road length} & \textbf{Regional} & \textbf{-3684.7} & \textbf{-5.2} & \textbf{0.078} \\
	\emph{Joint} & 4 & NPP & Regional & -3689.1 & -9.6 & 0.076 \\
	\emph{Joint} & 4 & Local veg. cover & Local & -3692.6 & -13.1 & 0.076 \\
	\emph{Joint} & 4 & H'(Land cover) & Regional & -3696.4 & -16.9 & 0.075 \\
	\emph{Joint} & 4 & Agriculture proportion & Regional & -3699.5 & -20.0 & 0.074 \\
	\emph{Joint} & 4 & Pasture/Crop & Local & -3704.6 & -25.1 & 0.073 \\
	\emph{Joint} & 4 & Edge proportion & Regional & -3711.2 & -31.7 & 0.071 \\
	\emph{Joint} & 4 & Building perimeter & Regional & -3717.1 & -37.6 & 0.069 \\
	\emph{Joint} & 4 & North aspect & Regional & -3720.8 & -41.3 & 0.069 \\
	\emph{Joint} & 4 & Canopy Type & Local & -3721.6 & -42.1 & 0.068 \\
	\emph{Joint} & 4 & Annual precipitation & Regional & -3724.9 & -45.4 & 0.067 \\
	\textbf{\emph{Joint}} & \textbf{5} & \textbf{Local veg. cover} & \textbf{Local} & \textbf{-3679.5} & \textbf{0.0} & \textbf{0.079} \\
	\emph{Joint} & 5 & Agriculture proportion & Regional & -3686.2 & -6.7 & 0.077 \\
	\emph{Joint} & 5 & NPP & Regional & -3690.3 & -10.8 & 0.076 \\
	\emph{Joint} & 5 & Annual precipitation & Regional & -3695.1 & -15.6 & 0.075 \\
	\emph{Joint} & 5 & H'(Land cover) & Regional & -3695.3 & -15.8 & 0.075 \\
	\emph{Joint} & 5 & Pasture/Crop & Local & -3698.2 & -18.7 & 0.074 \\
	\emph{Joint} & 5 & Building perimeter & Regional & -3703.7 & -24.2 & 0.073 \\
	\emph{Joint} & 5 & Edge proportion & Regional & -3705.0 & -25.5 & 0.072 \\
	\emph{Joint} & 5 & North aspect & Regional & -3707.7 & -28.2 & 0.072 \\
	\emph{Joint} & 5 & Canopy Type & Local & -3709.5 & -30.0 & 0.071 \\
	\emph{Joint} & 6 & H'(Land cover) & Regional & -3691.7 & -12.2 & 0.076 \\
	\emph{Joint} & 6 & Agriculture proportion & Regional & -3692.9 & -13.4 & 0.075 \\
	\emph{Joint} & 6 & Pasture/Crop & Local & -3694.5 & -15.0 & 0.075 \\
	\emph{Joint} & 6 & NPP & Regional & -3694.6 & -15.1 & 0.075 \\
	\emph{Joint} & 6 & Edge proportion & Regional & -3700.3 & -20.8 & 0.074 \\
	\emph{Joint} & 6 & Annual precipitation & Regional & -3703.0 & -23.5 & 0.073 \\
	\emph{Joint} & 6 & Building perimeter & Regional & -3710.1 & -30.6 & 0.071 \\
	\emph{Joint} & 6 & North aspect & Regional & -3712.0 & -32.5 & 0.071 \\
	\emph{Joint} & 6 & Canopy Type & Local & -3719.2 & -39.7 & 0.069 \\
	\emph{Joint} & 7 & Agriculture proportion & Regional & -3699.8 & -20.3 & 0.074 \\
	\emph{Joint} & 7 & NPP & Regional & -3702.6 & -23.1 & 0.073 \\
	\emph{Joint} & 7 & Annual precipitation & Regional & -3711.0 & -31.5 & 0.071 \\
	\emph{Joint} & 7 & Pasture/Crop & Local & -3721.3 & -41.8 & 0.068 \\
	\emph{Joint} & 7 & Building perimeter & Regional & -3722.3 & -42.8 & 0.068 \\
	\emph{Joint} & 7 & North aspect & Regional & -3724.3 & -44.8 & 0.068 \\
	\emph{Joint} & 7 & Canopy Type & Local & -3727.8 & -48.3 & 0.067 \\
	\emph{Joint} & 7 & Edge proportion & Regional & -3739.6 & -60.1 & 0.064 \\
	\emph{Joint} & 8 & NPP & Regional & -3709.0 & -29.5 & 0.071 \\
	\emph{Joint} & 8 & Annual precipitation & Regional & -3718.0 & -38.5 & 0.069 \\
	\emph{Joint} & 8 & Pasture/Crop & Local & -3722.6 & -43.1 & 0.068 \\
	\emph{Joint} & 8 & Building perimeter & Regional & -3727.5 & -48.0 & 0.067 \\
	\emph{Joint} & 8 & North aspect & Regional & -3732.7 & -53.2 & 0.066 \\
	\emph{Joint} & 8 & Canopy Type & Local & -3737.7 & -58.2 & 0.064 \\
	\emph{Joint} & 8 & Edge proportion & Regional & -3740.8 & -61.3 & 0.064 \\
	\emph{Joint} & 9 & Pasture/Crop & Local & -3729.5 & -50.0 & 0.066 \\
	\emph{Joint} & 9 & Annual precipitation & Regional & -3733.1 & -53.6 & 0.065 \\
	\emph{Joint} & 9 & North aspect & Regional & -3735.5 & -56.0 & 0.065 \\
	\emph{Joint} & 9 & Building perimeter & Regional & -3737.8 & -58.3 & 0.064 \\
	\emph{Joint} & 9 & Canopy Type & Local & -3753.0 & -73.5 & 0.06 \\
	\emph{Joint} & 9 & Edge proportion & Regional & -3778.5 & -99.0 & 0.054 \\
	\emph{Joint} & 10 & Annual precipitation & Regional & -3754.5 & -75.0 & 0.06 \\
	\emph{Joint} & 10 & Canopy Type & Local & -3755.9 & -76.4 & 0.06 \\
	\emph{Joint} & 10 & North aspect & Regional & -3758.7 & -79.2 & 0.059 \\
	\emph{Joint} & 10 & Building perimeter & Regional & -3761.7 & -82.2 & 0.058 \\
	\emph{Joint} & 10 & Edge proportion & Regional & -3800.2 & -120.7 & 0.049 \\
	\emph{Joint} & 11 & Canopy Type & Local & -3766.9 & -87.4 & 0.057 \\
	\emph{Joint} & 11 & North aspect & Regional & -3774.7 & -95.2 & 0.055 \\
	\emph{Joint} & 11 & Building perimeter & Regional & -3775.7 & -96.2 & 0.055 \\
	\emph{Joint} & 11 & Edge proportion & Regional & -3789.4 & -109.9 & 0.051 \\
	\emph{Joint} & 12 & North aspect & Regional & -3796.1 & -116.6 & 0.05 \\
	\emph{Joint} & 12 & Edge proportion & Regional & -3799.2 & -119.7 & 0.049 \\
	\emph{Joint} & 12 & Building perimeter & Regional & -3799.3 & -119.8 & 0.049 \\
	\emph{Joint} & 13 & Edge proportion & Regional & -3820.5 & -141.0 & 0.044 \\
	\emph{Joint} & 13 & Building perimeter & Regional & -3821.9 & -142.4 & 0.043 \\
	\emph{Joint} & 14 & Building perimeter & Regional & -3843.9 & -164.4 & 0.038 \\
\end{longtable}
\endgroup



\begin{figure}
	\centering\includegraphics[height=7.5in]{../../../ms/1_Ecography/1/figs/b_opt_byParam.png}
	\caption{\label{fig:b_byParam} Posterior species responses to local and regional covariates. Points show posterior medians, while lines show the Highest Posterior Density Intervals (HPDIs: 80\%: thick; 95\%: thin). Solid points indicate 95\% HPDIs that exclude zero in the \emph{Joint} (purple) and \emph{Structured} (green) models.}
\end{figure}

\begin{figure}
	\centering\includegraphics[width=6.5in]{../../../ms/1_Ecography/1/figs/b_opt_bar.png}
	\caption{\label{fig:b_bars} Proportion of species' responses to each covariate. Responses are considered positive (red) or negative (blue) if the Highest Posterior Density Interval (HPDI) excludes zero for (a) 95\% HPDIs and (b) 80\% HPDIs. }
\end{figure}

\begin{figure}
	\centering\includegraphics[height=8in]{../../../ms/1_Ecography/1/figs/lambda_zones.png}
	\caption{\label{fig:lam_zones} Posterior species intensity across plateau and montane zones in each model. Each species' total intensity (a. Regional; b. Local) was partitioned into the proportion occurring in plateau ($<$1000m: grey) and montane ($\geq$1000m: green) elevations. Vertical lines indicate 80\%, 90\%, and 95\% for either zone. }
\end{figure}

\begin{figure}
	\centering\includegraphics[width=5in]{../../../ms/1_Ecography/1/figs/DPCoA_Struc.png}
	\caption{\label{fig:dpcoa_struc} Double principle coordinate analysis (DPCoA) of local communities predicted by the \emph{Structured} model, with plots colored by (a) elevational bin, and (b) region. The central plateau includes hills from $\sim$300m to $\sim$1000m, with the Jura and the Alps rising steeply in the east and west, respectively. }
\end{figure}

\begin{figure}
	\centering\includegraphics[height=8in]{../../../ms/1_Ecography/1/figs/D.png}
	\caption{\label{fig:D} Proportional taxonomic bias posterior distributions. Medians and Highest Posterior Density Intervals (HPDIs; 80\%: thick; 95\%: thin) for the representation of each species in the presence-only dataset relative to the structured abundance dataset. Values less than 1 indicate under-representation in the presence-only dataset. Dark red: 95\% HPDIs exclude zero; light red: 80\% HPDIs exclude zero. }
\end{figure}

\begin{figure}
	\centering\includegraphics[width=5in]{../../../ms/1_Ecography/1/figs/maps/sf_S_J.png}
	\caption{\label{fig:sf_map_Joint} Maps of predicted richness in the \emph{Joint} model for subfamilies. Predictions are based on 95\% Highest Posterior Density Intervals for each species. }
\end{figure}


\begin{figure}
	\centering\includegraphics[width=5in]{../../../ms/1_Ecography/1/figs/maps/sf_S_S.png}
	\caption{\label{fig:sf_map_Structured} Maps of predicted richness in the \emph{Structured} model for subfamilies. Predictions are based on 95\% Highest Posterior Density Intervals for each species. }
\end{figure}

\begin{figure}
	\centering\includegraphics[width=6in]{../../../ms/1_Ecography/1/figs/maps/gen_S_J.png}
	\caption{\label{fig:gen_map_Joint} Maps of predicted richness in the \emph{Joint} model for genera with at least two species. Predictions are based on 95\% Highest Posterior Density Intervals for each species. }
\end{figure}

\begin{figure}
	\centering\includegraphics[width=6in]{../../../ms/1_Ecography/1/figs/maps/gen_S_S.png}
	\caption{\label{fig:gen_map_Structured} Maps of predicted richness in the \emph{Structured} model for genera with at least two species. Predictions are based on 95\% Highest Posterior Density Intervals for each species. }
\end{figure}







\end{document}
